proto/dymension/delayedack/rollapp_packet.proto

```diff
@@ -8,14 +8,22 @@ option go_package = "github.com/dymensionxyz/dymension/x/delayedack/types";
 
 message RollappPacket {
     ibc.core.channel.v1.Packet packet = 1;
+    bytes acknowledgement = 2;
+    bool isTimeout = 3;
```

Comment:
not used

Reply:
the `isTimeout` flag not used as I see it. 
timeout is set by the type (`RollappPacket_ON_TIMEOUT`)

Reply:
True! not necessary.